lib/matplotlib/tests/test_subplots.py

```diff
@@ -40,6 +41,20 @@ def check_visible(axs, x_visible, y_visible):
             assert ax.get_ylabel() == ""
 
 
+def check_tick1_visible(axs, x_visible, y_visible):
+    """
+    Check that the x and y tick visibility is as specified.
+
+    Note: This only checks the tick1line, i.e. bottom / left ticks.
+    """
+    for ax, visible, in zip(axs, x_visible):
+        for tick in ax.xaxis.get_major_ticks():
+            assert tick.tick1line.get_visible() == visible
+    for ax, y_visible, in zip(axs, y_visible):
+        for tick in ax.yaxis.get_major_ticks():
+            assert tick.tick1line.get_visible() == visible
```

Comment:
I am very confused by coverage claiming this line is not run, the only thing that makes sense is if in every case where we test this `ax.yaxis.get_major_ticks()` is an empty list?

Do we need to force a draw at the top of this checker?